docker-compose.yml

```diff
@@ -24,10 +24,8 @@ services:
 #   Uncomment the following lines to mount workspaceConfig file
 #   volumes:
 #     - <absolute_path_to_workspace_config>:/etc/rudderstack/workspaceConfig.json
-  d-transformer:
-    build:
-      context: ./rudder-transformer/
-      dockerfile: Dockerfile
+  transformer:
+    image: "rudderlabs/develop-rudder-transformer:master"
```

Comment:
with this change can we also completely get rid of the transformer submodule in the rudder-server repo right?

Reply:
Yeah, I think it makes more sense from rudder-server to use the docker image.

Not sure if it create inconvenience for people working with transformer